### ISSUE TITLE
Gear overlay over profile icon is a bit strange

### ISSUE THREAD

**Issue by bpasero** [microsoft/vscode#194194]
I wonder if we should not overlay here:

![Image](https://github.com/microsoft/vscode/assets/900690/1d0cbc12-d034-4d94-a2c8-9ea0c2d32a31)

My understanding is that this is the only place where we show the profile icon but then its always overlayed with the gear which is quite large.

**Comment by gjsjohnmurray** [comment]
I tend to agree that the overlay is superfluous, but is an underlay possible as an alternative to be tried?

**Comment by sandy081** [comment]
We are trying to come up with a smaller gear icon for overlay

![image](https://github.com/microsoft/vscode/assets/10746682/f2e90cc4-774e-4aa8-827d-93e5a8589d0e)

![image](https://github.com/microsoft/vscode/assets/10746682/7f341a84-861b-4fc6-8642-e9d7d6cfec05)

I am preferring to have gear overlay icon to indicate it is for settings, this will be helpful when there are duplicates like extensions profile icon.

**Comment by sandy081** [comment]
I am thinking to undo the overlay until we have better overlay icon.
